xs2a-adapter-service-impl/src/main/java/de/adorsys/xs2a/adapter/impl/http/wiremock/WiremockFileResolver.java

```diff
@@ -26,28 +26,36 @@ public boolean check(String url, String method, String body) {
         }
 
     },
-    AIS_AUTHORISE_PSU("ais-authorise-psu.json") {
+    AIS_START_PSU_AUTHENTICATION("ais-start-psu-authentication.json") {
         @Override
         public boolean check(String url, String method, String body) {
             return PUT_METHOD.equalsIgnoreCase(method)
```

Comment:
I think start authorization always required the POST method

Reply:
Good spot, fixed.
TY